### PR TITLE
feat: add missing utility classes to LumoUtility

### PR DESCRIPTION
**Pull request by anezthes**
## Description

Added missing utility classes to `LumoUtility`.

Fixes # (issue)

## Type of change

- [x] Bugfix
- [ ] Feature

## Checklist

- [x] I have read the contribution guide: https://vaadin.com/docs/latest/contributing/overview
- [x] I have added a description following the guideline.
- [ ] The issue is created in the corresponding repository and I have referenced it.
- [ ] I have added tests to ensure my change is effective and works as intended.
- [x] New and existing tests are passing locally with my change.
- [x] I have performed self-review and corrected misspellings.

### CODE REVIEW

File: vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java [hunk 1]
```diff
@@ -92,6 +92,36 @@ private AlignSelf() {
 
     }
 
+    /**
+     * Classes for setting the backdrop blur of an element.
```
Comment (web-padawan) [inline vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java:96]:
```suggestion
     * Classes for setting the aspect ratio of an element.
```

File: vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java [hunk 2]
```diff
@@ -319,29 +403,48 @@ private XLarge() {
...
 
-        public static final String GROW = "flex-grow";
-        public static final String GROW_NONE = "flex-grow-0";
+        public static final String GROW = "grow";
+        public static final String GROW_NONE = "grow-0";
 
-        public static final String SHRINK = "flex-shrink";
-        public static final String SHRINK_NONE = "flex-shrink-0";
+        public static final String SHRINK = "shrink";
+        public static final String SHRINK_NONE = "shrink-0";
```
Comment (web-padawan) [inline vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java:447]: I don't see these changes in the web components repo. Also, it would be better to extract renaming classes into separate PR (as it's changing the existing classes, not related to adding missing classes).

File: vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java [hunk 3]
```diff
@@ -846,26 +948,23 @@ public static final class XXLarge {
...
-
     }
 
     /**
      * Classes for defining the height of an element.
      */
     public static final class Height {
 
-        public static final String AUTO = "h-auto";
-        public static final String FULL = "h-full";
```
Comment (web-padawan) [inline vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java:860]: Some changes here are just about moving rows around. IMO it's better to revert those for smaller diff.

File: vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java (hunk 2 above)
Comment (web-padawan) [reply vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java:447]: UPD: web components classes renaming is done in https://github.com/vaadin/web-components/pull/7515. I still think we should extract change from `flex-grow` to `grow` etc into separate PR at least to have a separate changelog entry.

File: vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java (hunk 3 above)
Comment (anezthes) [reply vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java:860]: Yeah, I want them to line up with the utility classes. Easier to keep track.

File: vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java (hunk 2 above)
Comment (anezthes) [reply vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java:447]: I'll make a separate PR.

**Review by web-padawan — APPROVED** [review]
